Tests/Cells/BaseBrickCellTests.swift

```diff
@@ -182,6 +182,44 @@ class BaseBrickCellTests: XCTestCase {
         XCTAssertNotNil(cell3?.contentView.viewWithTag(21))
         XCTAssertEqual(cell3?.contentView.subviews.first, backgroundView1)
     }
+    
+    func testBrickViewAppearance() {
+        brickView.registerBrickClass(LabelBrick.self)
+        
+        let testLabelBrick = LabelBrick("AppearanceTest", height: .Fixed(size: 100), text: "Appearance Test")
+        testLabelBrick.brickCellAppearanceDataSource = self
+        
+        brickView.setSection(BrickSection(bricks: [testLabelBrick]))
+        brickView.layoutSubviews()
+        
+        guard let cell = brickView.cellForItemAtIndexPath(NSIndexPath(forItem: 0, inSection: 1)) as? BrickCell else {
+            XCTAssert(false, "Cell Should Not be nil")
+            return
+        }
+        
+        cell.updateBrickCell(for: .Loading)
```

Comment:
This test is to see if the function works, but where is this executed on the BrickCollectionView?